src/app/parcels/ParcelsPage.tsx

```diff
@@ -358,6 +359,27 @@ const ParcelsPage: React.FC<{}> = () => {
         }
     }, [parcelId]);
 
+    useEffect(() => {
+        const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+            const { data, error } = await supabase
+                .from("parcels")
+                .select("client_id")
+                .eq("primary_key", parcelId)
+                .single();
+
+            if (error) {
+                void logError("...");
+                return;
+            }
+
+                const fetchedClientId = data.client_id;
+                setClientIdForSelectedParcel(fetchedClientId);
```

Comment:
I don't think this is indented correctly?

Reply:
Amended